### ISSUE TITLE
[UPDATE] cert-manager apiVersion and acme files

### ISSUE THREAD

**Issue by gentios** [hyperledger-labs/nephos#222]
Hi everyone, I tried to deploy the `examples/qa` in a real cluster and I struggled with some changes of `Kubernetes v1.16.2`.

I encountered the following issues: 

The following command `helm install stable/cert-manager -n cert-manager --namespace cert-manager` does no longer work since `cert-manager@0.12` requires some additional repos to be added in helm.

Also their`apiVersion` has changed from: `certmanager.k8s.io/v1alpha1` to `cert-manager.io/v1alpha2` 

Their acme resolvers now require additional solvers fields etc..

I encountered these issues and I managed to fix them, I would be happy to create a PR regarding this if you agree.

**Comment by alexvicegrab** [comment]
@gentios, yes, you are correct. It is necessary to update the installation and configuration of the Certificate Manager.

PRs on this are welcome. Unfortunately, I do not have time to help contribute to this.
